src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/main.js

```diff
@@ -101,6 +100,33 @@ define([
         },
 
         /**
+         * Creates and returns the breadcrumb object, from the current data.
+         *
+         * @returns {Object} the breadcrumb data
+         */
+        getBreadcrumb: function() {
+            if (!this.options.id) {
+                return [];
+            }
+
+            var breadcrumb = [{
+                title: 'sulu.media.all',
+                data: {}
+            }];
+
+            this.data._embedded.breadcrumb.forEach(function(crumb) {
```

Comment:
Why not calling the `breadcrumb` variables `breadcrumbs`, so that the `crumb` variable here can be renamed to `breadcrumb`?